### DISCUSSION TITLE
Congelar versión de Cotalker en infraestructura dedicada

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#38] [3. Ideas & Feature Requests] [answered]
Se requiere dejar al cliente REDACTED (@Yiakman) en versión estable de Cotalker y sacar del ci/cd para priorizar estabilidad vs nuevas features. Además algunos usuarios no entienden funcionalidad de nuevos botones en su flujo.

**Comment by acedward** [comment, marked as answer]
@jdurandpaz si es un feature request que se requiere por concepto de negocio, horas vendidas por favor pedir por interno - este foro solo es para discutir las funcionalidades públicas del producto.

Por este medio no estamos asegurando ningún SLA de servicio.

**Reply by acedward** [reply]
Si la pregunta, es ¿En infraestructura cloud dedica, es posible freezar una version estable?
La respuesta es, si - absolutamente, pero es necesario contar con un contrato de soporte enterprise.

**Reply by jdurandpaz** [reply]
OK gracias
